src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  margin: 0 auto;
+  width: $calendar-width;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: left;
+  align-items: center;
+  gap: $day-gap;
+  padding: $calendar-padding;
+
+  &__day {
+    display: none;
+    height: 100px;
+    width: $day-width;
+    background-color: $background-color;
+    border: 1px solid $border-color;
+    transition: all 0.5s;
```

Comment:
It is better not to set transitions for everything, it affects performance. it's better to be specific.